.gitpod/start_repo.sh

```diff
@@ -11,29 +11,26 @@ DEFAULT_ARTIFACTS="${DDEV_REPO}-artifacts"
 export DDEV_ARTIFACTS=${DDEV_ARTIFACTS:-$DEFAULT_ARTIFACTS}
 echo "Attempting to get artifacts from ${DDEV_ARTIFACTS}"
 git clone ${DDEV_ARTIFACTS} "/tmp/${DDEV_ARTIFACTS##*/}" || echo "Could not check out artifacts repo ${DDEV_ARTIFACTS}"
-reponame=${DDEV_REPO##*/}
-reponame="${reponame//_/-}"
-git clone ${DDEV_REPO} ${GITPOD_REPO_ROOT}/${reponame}
-if [ -d ${GITPOD_REPO_ROOT}/${reponame} ]; then
-  "$GITPOD_REPO_ROOT"/.gitpod/setup_vscode_git.sh
-  cd ${GITPOD_REPO_ROOT}/${reponame}
+REPO_NAME=${DDEV_REPO##*/}
+export REPO_NAME="${REPO_NAME//_/-}"
+REPO_HOME=/workspace/code
+git clone ${DDEV_REPO} ${REPO_HOME}
+if [ -d ${REPO_HOME} ]; then
```

Comment:
You shouldn't have to do this, since we just created and cloned into it. And we have `set -eu` at the top, meaning it would failed the `mkdir` and the `git clone`.

Reply:
You mean remove the `if` indentation?

Reply:
Yes, just saying that directory has to exist by this point. Just remove the if and endif. But I'm not insisting, it doesn't do harm, just adds complexity for the next person that looks at it.

Reply:
Done.